connectors/src/connectors/zendesk/lib/zendesk_api.ts

```diff
@@ -0,0 +1,11 @@
+import { createClient } from "node-zendesk";
+
+export function createZendeskClient({
+  token,
+  subdomain = "d3v-dust",
```

Comment:
What's d3v-dust?

Reply:
it's the subdomain we use for tests, in a future work we'll fix the authentication flow to retrieve the client's subdomain and use it here